packages/amplify-category-api/src/commands/api/add-graphql-datasource.ts

```diff
@@ -121,7 +121,7 @@ export const run = async (context: $TSContext): Promise<void> => {
 
       fs.writeFileSync(graphqlSchemaFilePath, graphql.print(concatGraphQLSchemaDoc), 'utf8');
     } else if (fs.existsSync(schemaDirectoryPath)) {
-      const rdsSchemaFilePath = path.join(schemaDirectoryPath, 'rds.graphql');
+      const rdsSchemaFilePath = path.join(schemaDirectoryPath, 'sql.graphql');
 
       fs.writeFileSync(rdsSchemaFilePath, graphql.print(rdsGraphQLSchemaDoc), 'utf8');
     } else {
```

Comment:
nit: The line 130 still mentions about `RDS schema` which can be `SQL schema` according to the PR's theme.

Reply:
Thanks for catching. Will follow up in a subsequent PR, so as not to dismiss these reviews & e2e test results (assuming they pass)